Java/libraries/recognizers-text/src/main/java/com/microsoft/recognizers/text/ExtendedModelResult.java

```diff
@@ -1,19 +0,0 @@
-package com.microsoft.recognizers.text;
-
-import java.util.SortedMap;
-
-public class ExtendedModelResult extends ModelResult {
```

Comment:
The previous design was a bit more "clear" as the extra property would only appear in very specific cases not enabled by default. But if this is not necessary to avoid the null problem on serializaition, thanks for deleting. 
As long as the old API surface on ModelResult remains the same (i.e., no breaking change), it should be fine.